shared_model/utils/string_builder.hpp

```diff
@@ -62,6 +62,25 @@ namespace shared_model {
         return *this;
       }
 
+      /**
+       * Appends a new named collection to string
+       * @tparam Collection - type of collection
+       * @tparam Transform - type of transformation function
+       * @param name - field name to append
+       * @param c - collection to append
+       * @param t - transformation function
+       */
+      template <typename Collection, typename Transform>
+      PrettyStringBuilder &appendAll(
```

Comment:
Can't these overloads of `appendAll` be merged into one function by giving the `name` argument default value of empty string and checking it in the body?